stubs/oauthlib/oauthlib/oauth1/rfc5849/__init__.pyi

```diff
@@ -56,4 +56,11 @@ class Client:
     ): ...
     def get_oauth_signature(self, request): ...
     def get_oauth_params(self, request): ...
-    def sign(self, uri, http_method: str = "GET", body: str | None = None, headers: dict[str, str] | None = None, realm=None): ...
+    def sign(
+        self,
+        uri,
+        http_method: str = "GET",
```

Comment:
you can import `Literal` from common stubs: `from oauthlib.common import _HTTPMethod` to indicate what kind of strings these are:
```suggestion
        http_method: _HTTPMethod = "GET",
```